### PR TITLE
done with movie card design

### PR DESCRIPTION
**Pull request by kidmeeno**

### CODE REVIEW

**Comment by daniellamarr** [comment]
Please provide screenshots

**Comment by kidmeeno** [comment]
moviecards

File: src/components/starRating/starRating.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React,{Component} from 'react';
+import './starRating.scss'
+class StarRating extends Component {
```
Comment (timzprof) [inline src/components/starRating/starRating.js]: How will the rating be made dynamic?? 
Cause you'll be getting values like 3.5, 4.5 for rating from the API

File: src/pages/homePage/homePage.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './homePage.scss';
+import MovieCard from '../../components/movieCard/movieCard';
+
+const HomePage = () => {
```
Comment (timzprof) [inline src/pages/homePage/homePage.js]: How about the Home Component, what's the difference between this and that component??

**Pull request closed without merging**